helper/testhelpers/postgresql/postgresqlhelper.go

```diff
@@ -12,44 +12,78 @@ import (
 )
 
 func PrepareTestContainer(t *testing.T, version string) (func(), string) {
-	return prepareTestContainer(t, version, "secret", "database")
+	env := []string{
+		"POSTGRES_PASSWORD=secret",
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, "secret", "database", true, false, false, env)
+
+	return cleanup, url
 }
 
 func PrepareTestContainerWithPassword(t *testing.T, version, password string) (func(), string) {
-	return prepareTestContainer(t, version, password, "database")
+	env := []string{
+		"POSTGRES_PASSWORD=" + password,
+		"POSTGRES_DB=database",
+	}
+
+	_, cleanup, url, _ := prepareTestContainer(t, "postgres", "postgres", version, password, "database", true, false, false, env)
+
+	return cleanup, url
+}
+
+func PrepareTestContainerRepmgr(t *testing.T, name, version string, envVars []string) (*docker.Runner, func(), string, string) {
+	return prepareTestContainer(t, name, "bitnami/postgresql-repmgr", version, "secret", "", false, true, true, envVars)
+}
+
+func StopContainer(t *testing.T, ctx context.Context, runner *docker.Runner, containerID string) {
+	if err := runner.Stop(ctx, containerID); err != nil {
+		t.Fatalf("Could not stop docker Postgres: %s", err)
+	}
+}
+
+func RestartContainer(t *testing.T, ctx context.Context, runner *docker.Runner, containerID string) {
+	if err := runner.Restart(ctx, containerID); err != nil {
+		t.Fatalf("Could not restart docker Postgres: %s", err)
+	}
 }
 
-func prepareTestContainer(t *testing.T, version, password, db string) (func(), string) {
+func prepareTestContainer(t *testing.T, name, repo, version, password, db string, addSuffix, forceLocalAddr, doNotAutoRemove bool, envVars []string) (*docker.Runner, func(), string, string) {
```

Comment:
Can remove `db` are here since it's been lifted into the env vars. Appears unused.